content/projects/gsoc/students.adoc

```diff
@@ -50,31 +81,205 @@ We highly recommend to make some contributions to the project while you work on
 It will help you to polish the proposal,
 and mentors will consider contributions and interactions with the community when processing applications.
 
-Useful links:
+[[UsefulLinks]]
+=== Useful links
+
+Here are a list of links to help you get started on participating in Jenkins and in coding for the Jenkins project,
+in increasing level of difficulty.
+
+* link:/project/governance/[Project Governance]
+* link:/participate[Participate and Contribute]
+* link:https://wiki.jenkins.io/display/JENKINS/Plugin+tutorial[Plugin tutorial]
+* link:/blog/2017/08/07/intro-to-plugin-development/[Plugin Development Tutorials, Videos, and More]
+
+Some older tutorials (warning, some may be a bit out of date):
```

Comment:
Linked from the blogpost above, maybe an overkill

Reply:
I removed this mini-section.